changelog/63601.added.md

```diff
@@ -0,0 +1 @@
+systemd daemon is reloaded, added in service state and module docs.
```

Comment:
This seems different from the changes to the docs

Reply:
Yes, sorry I add this commit into this PR by mistake, It is from a different PR.

Reply:
Yes, sorry I added this commit to this PR by mistake, it was from a different PR.